lib/repl.js

```diff
@@ -421,7 +418,7 @@ function REPLServer(prompt,
       wrappedCmd = true;
     }
 
-    if (experimentalREPLAwait && StringPrototypeIncludes(code, 'await')) {
```

Comment:
We have now a mechanism to flip the default value of a flag (making it possible to opt out with `--no-experimental-repl-await`.
Are we confident enough about the feature to remove the flag?

Reply:
That does sound like a good way to ensure an opt-out in case there are issues.

Reply:
So, do we introduce `--no-experimental-repl-await` flag and check if it not set here or we use the `ImpliesNot("--no-harmony-top-level-await", "--experimental-top-level-await");`?  Either way we would need to `getOptionValue` and check if it is not set in this condition, right?

Reply:
@hemanth I believe just the `ImpliesNot` line is enough here, and then changing the default value for `--experimental-repl-await` to being enabled. That should fully cover the feature per my reading of https://github.com/nodejs/node/pull/39023.

Reply:
Cool, so `experimental-repl-await` shall be undocumented and enabled, `--no-experimental-repl-await` shall be documented.

Reply:
Thanks for making the change! I added a few suggestions